website/versioned_docs/version-v1.x/integrating.md

```diff
@@ -11,6 +11,91 @@ MACI can be used in any protocol that requires collusion resistance, for instanc
 
 Here we will be looking at QFI and how it was used. Please note that this will be expanded as QFI is updated to use the newest version of MACI. Should you decide to integrate MACI in the meantime, feel free to open an issue on the GitHub repo.
 
+## Deployment
+
+First, you need to deploy contracts to start using MACI. This can be easily done by `maci-cli`.
+Deployment order is:
+
+1. Deploy crypto (Hasher, Poseidon)
+2. Deploy VK Registry
+3. Set verification keys
+4. Deploy VoiceCreditProxy
+5. Deploy Gatekeeper
+6. Deploy Verifier
+7. Deploy Topup credit
+8. Deploy MessageProcessorFactory, PollFactory, SubsidyFactory, TallyFactory
+9. Deploy MACI, AccQueueQuinaryMaci
+10. Deploy Poll, AccQueueQuinaryMaci, MessageProcessor, Tally and Susbsidy (optional)
+
+Before running the deploy command make sure you have [zkey files](https://maci.pse.dev/docs/trusted-setup) and env variables `ETH_PROVIDER` (RPC endpoint) and `ETH_SK` (wallet private key) are set.
```

Comment:
Let's add a line to explain that the ones in your examples are not safe to use in prod (only the ones in docs/trusted-seutp are)